dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/KyuubiSparkSQLExtension.scala

```diff
@@ -20,8 +20,7 @@ package org.apache.kyuubi.sql
 import org.apache.spark.sql.SparkSessionExtensions
 
 import org.apache.kyuubi.sql.sqlclassification.KyuubiSqlClassification
-import org.apache.kyuubi.sql.zorder.ResolveZorder
-import org.apache.kyuubi.sql.zorder.ZorderSparkSqlExtensionsParser
+import org.apache.kyuubi.sql.zorder.{InsertZorderBeforeWritingDatasource, InsertZorderBeforeWritingHive, ResolveZorder, ZorderSparkSqlExtensionsParser}
```

Comment:
this line length exceeds 100 characters.

Reply:
it's ok in `import`, see the scalastyle configuration
```
  <check level="error" class="org.scalastyle.file.FileLineLengthChecker" enabled="true">
    <parameters>
      <parameter name="maxLineLength"><![CDATA[100]]></parameter>
      <parameter name="tabSize"><![CDATA[2]]></parameter>
      <parameter name="ignoreImports">true</parameter>
    </parameters>
  </check>
```

Reply:
btw, my Intellij IDEA always can't work, how to solve it? thanks.

![image](https://user-images.githubusercontent.com/86483005/133030586-235d8e6d-f869-4c59-a03f-32842879b367.png)

Reply:
I think it's your JDK version issue,  the following is my JDK

![image](https://user-images.githubusercontent.com/12025282/133031046-db4f92d9-7186-4e20-be6c-31dbac7a54bc.png)

Reply:
ok, will try it